src/api/Areas/Dashboard/Controllers/ConfigurationItemController.cs

```diff
@@ -0,0 +1,116 @@
+{
+    #region Variables
+    private readonly ILogger _logger;
+    private readonly IConfigurationItemService _service;
+    private readonly IXlsExporter _exporter;
+    #endregion
+
+    #region Constructors
+    /// <summary>
+    /// Creates a new instance of a ConfigurationItemController.
+    /// </summary>
+    /// <param name="service"></param>
+    /// <param name="exporter"></param>
+    /// <param name="logger"></param>
+    public ConfigurationItemController(IConfigurationItemService service, IXlsExporter exporter, ILogger<ConfigurationItemController> logger)
+    {
+        _service = service;
+        _exporter = exporter;
+        _logger = logger;
+    }
+    #endregion
+
+    #region Endpoints
+    // TODO: Limit based on role and tenant.
+    /// <summary>
+    ///
+    /// </summary>
+    /// <returns></returns>
+    [HttpGet(Name = "GetConfigurationItems-Dashboard")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(IEnumerable<ConfigurationItemModel>), (int)HttpStatusCode.OK)]
+    [SwaggerOperation(Tags = new[] { "ConfigurationItem" })]
+    public IActionResult Get()
+    {
+        var uri = new Uri(this.Request.GetDisplayUrl());
+        var query = Microsoft.AspNetCore.WebUtilities.QueryHelpers.ParseQuery(uri.Query);
+        var filter = new HSB.Models.Filters.ConfigurationItemFilter(query);
+        var result = _service.Find(filter.GeneratePredicate(), filter.Sort);
+        return new JsonResult(result.Select(ci => new ConfigurationItemModel(ci)));
+    }
+
+    // TODO: Limit based on role and tenant.
+    /// <summary>
+    ///
+    /// </summary>
+    /// <param name="id"></param>
+    /// <returns></returns>
+    [HttpGet("{id}", Name = "GetConfigurationItem-Dashboard")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(ConfigurationItemModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType((int)HttpStatusCode.NoContent)]
+    [SwaggerOperation(Tags = new[] { "ConfigurationItem" })]
+    public IActionResult GetForId(int id)
+    {
+        var entity = _service.FindForId(id);
+
+        if (entity == null) return new NoContentResult();
+
+        return new JsonResult(new ConfigurationItemModel(entity));
+    }
+
+    // TODO: Complete functionality
+    // TODO: Limit based on role and tenant.
+    /// <summary>
+    ///
+    /// </summary>
+    /// <param name="format"></param>
+    /// <param name="name"></param>
+    /// <returns></returns>
+    /// <exception cref="NotImplementedException"></exception>
+    [HttpGet("export")]
+    [Produces("application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")]
+    [ProducesResponseType((int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "ConfigurationItem" })]
+    public IActionResult Export(string format, string name = "service-now")
```

Comment:
Still need to do a lot of work with the Excel export.  Will most likely create a separate story/task for this.